python/python-core/intro-to-modules/what-are-modules.md

```diff
@@ -7,11 +7,13 @@ links:
 practiceQuestion:
   formats:
     - fill-in-the-gap
-  context: relative
+    - type-in-the-gap
```

Comment:
they could answer `py` which is correct and that fails; not a good question for type
```suggestion
```